redirects.yaml

```
@@ -1,2 +1,2 @@
-support: /enterprise-support
-enterprise-support/plans-and-pricing: /pricing
+support: /pro
```

I meant this, but the support one was needed too 🙂 (if it's not included the users will get [404](https://jp-ubuntu-com-489.demos.haus/enterprise-support) if they try to visit the old url)
```suggestion
enterprise-support: /pro
support: /pro
```